diff --git a/av/video/frame.pyx b/av/video/frame.pyx
index ee060e1..a7c794a 100644
--- a/av/video/frame.pyx
+++ b/av/video/frame.pyx
@@ -591,6 +591,7 @@ cdef class VideoFrame(Frame):
 
         return frame
 
+    @staticmethod
     def from_bytes(img_bytes: bytes, width: int, height: int, format="rgba", flip_horizontal=False, flip_vertical=False):
         frame = VideoFrame(width, height, format)
         if format == "rgba":
